src/OpenFeature.Contrib.Providers.ConfigCat/OpenFeature.Contrib.Providers.ConfigCat.csproj

```diff
@@ -16,6 +16,6 @@
     </AssemblyAttribute>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="ConfigCat.Client" Version="[9,)"/>
+    <PackageReference Include="ConfigCat.Client" Version="9.2.0"/>
```

Comment:
Could a range be used instead of a specific version?

Reply:
.NET considers the single version to be equivalent to be a range. 

The following are equivalent:
```xml
    <PackageReference Include="ConfigCat.Client" Version="9.2.0"/>
    <PackageReference Include="ConfigCat.Client" Version="[9.2.0, )"/>
```

Reply:
For reference: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges 


Notation | Applied rule | Description
-- | -- | --
1.0 | x ≥ 1.0 | Minimum version, inclusive
[1.0,) | x ≥ 1.0 | Minimum version, inclusive

Reply:
Yep, as shown in [this table](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges), the two are equivalent. I would go with the simpler syntax.

Reply:
Got it, thanks.